src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantOverlapAnnotator.java

```diff
@@ -28,7 +28,7 @@
      * @see #VariantOverlapAnnotator(org.broadinstitute.gatk.utils.commandline.RodBinding, java.util.Map, org.broadinstitute.gatk.utils.GenomeLocParser)
      */
     public VariantOverlapAnnotator(final FeatureInput<VariantContext> dbSNPBinding) {
```

Comment:
Wow, how did this class get merged in this terrible state. The documentation is awful, and refers to GATK3 classes and concepts that no longer exist in GATK4. `RefMetaDataTracker`? `RodBinding`? Can you please clean this up?